libs/wingsdk/src/std/map.ts

```diff
@@ -165,7 +179,8 @@ export class MutMap {
    * @macro ($self$)[$args$]
    *
    * @param key The key of the element to return.
-   * @returns The element associated with the specified key, or undefined if the key can't be found
+   * @returns The element associated with the specified key
+   * @throws If the key can't be found
```

Comment:
Need to create a macro here to throw if not found (just like above)

Reply:
Yea it would be great, but unfortunately today we have a few limitations. (Primarily type unions if I recall right)

Reply:
Done! [`38ae38e` (#3771)](https://github.com/winglang/wing/pull/3771/commits/38ae38e8687ad10c502b010a52c8fb6e25fad7e3)